diff --git a/output/resources/ars_additions/.cache/0567507aacc716a599c12d342edeb1f0d5e76f02 b/output/resources/ars_additions/.cache/0567507aacc716a599c12d342edeb1f0d5e76f02
index 1464c3f..6ab0fea 100644
--- a/output/resources/ars_additions/.cache/0567507aacc716a599c12d342edeb1f0d5e76f02
+++ b/output/resources/ars_additions/.cache/0567507aacc716a599c12d342edeb1f0d5e76f02
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.426636494	Imbuement Charging Datagen
+// 1.21.1	2025-08-24T22:23:42.978498623	Imbuement Charging Datagen
 4e8554883edc2ccfd98f8351fa588c18fc4ae428 data/ars_additions/recipe/imbuement_charging/dispel_protection_charm.json
 1f8126e4d3ad7146644cd1a3e2bcf654f2f845a3 data/ars_additions/recipe/imbuement_charging/ender_mask_charm.json
 d6b6ccfc2d2c6d010bf7aaae7d28e81795bbd024 data/ars_additions/recipe/imbuement_charging/fall_prevention_charm.json
diff --git a/output/resources/ars_additions/.cache/083d3595094bab29f0c8e1482a6475d4fb5b5300 b/output/resources/ars_additions/.cache/083d3595094bab29f0c8e1482a6475d4fb5b5300
index 34c0c22..5052056 100644
--- a/output/resources/ars_additions/.cache/083d3595094bab29f0c8e1482a6475d4fb5b5300
+++ b/output/resources/ars_additions/.cache/083d3595094bab29f0c8e1482a6475d4fb5b5300
@@ -1,3 +1,3 @@
-// 1.21.1	2025-08-24T22:07:42.438716578	Processor Lists
+// 1.21.1	2025-08-24T22:23:42.99027324	Processor Lists
 40a769c1f16511f4d3c9097ce150112944bba367 data/ars_additions/worldgen/processor_list/arcane_library.json
 c47810331fb85dbbe6140a10cebdc14c421ca211 data/ars_additions/worldgen/processor_list/nexus_tower.json
diff --git a/output/resources/ars_additions/.cache/0c301f81b47090d78ddc75e103683bca025fd3f8 b/output/resources/ars_additions/.cache/0c301f81b47090d78ddc75e103683bca025fd3f8
index be3e7d4..600c4b8 100644
--- a/output/resources/ars_additions/.cache/0c301f81b47090d78ddc75e103683bca025fd3f8
+++ b/output/resources/ars_additions/.cache/0c301f81b47090d78ddc75e103683bca025fd3f8
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.435450957	Tags for minecraft:block mod id ars_additions
+// 1.21.1	2025-08-24T22:23:42.990052278	Tags for minecraft:block mod id ars_additions
 df3ceb369835c6795542dc4d94e20ba51c8e88ff data/ars_nouveau/tags/block/an_decorative.json
 f324a130d6d8683502003a130d09fc99c221670b data/c/tags/block/relocation_not_supported.json
 f324a130d6d8683502003a130d09fc99c221670b data/carryon/tags/block/block_blacklist.json
diff --git a/output/resources/ars_additions/.cache/103d9f3f36b01595f1aa5172191e60eff02e6924 b/output/resources/ars_additions/.cache/103d9f3f36b01595f1aa5172191e60eff02e6924
index ff34cd0..8a51ca1 100644
--- a/output/resources/ars_additions/.cache/103d9f3f36b01595f1aa5172191e60eff02e6924
+++ b/output/resources/ars_additions/.cache/103d9f3f36b01595f1aa5172191e60eff02e6924
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:07:42.440046739	Registries
+// 1.21.1	2025-08-24T22:23:42.991778325	Registries
 a6b1e5d8eaf9c20e90cf46ffff1ef6fcede0d6c4 data/ars_additions/enchantment/spellweave.json
diff --git a/output/resources/ars_additions/.cache/1f3b90a25c3d378e4643ad9c67cde9e06e2af0a9 b/output/resources/ars_additions/.cache/1f3b90a25c3d378e4643ad9c67cde9e06e2af0a9
index db5ea21..88fae3e 100644
--- a/output/resources/ars_additions/.cache/1f3b90a25c3d378e4643ad9c67cde9e06e2af0a9
+++ b/output/resources/ars_additions/.cache/1f3b90a25c3d378e4643ad9c67cde9e06e2af0a9
@@ -1,3 +1,3 @@
-// 1.21.1	2025-08-24T22:07:42.435118237	Tags for minecraft:entity_type mod id ars_additions
+// 1.21.1	2025-08-24T22:23:42.989665307	Tags for minecraft:entity_type mod id ars_additions
 b4f83f9a6515971ba7861d116d4ddaa339347d5d data/ars_additions/tags/entity_type/source_spawner_denylist.json
 35133e95f1c8fdd7a1c21afcc231fc0bffefb9a8 data/ars_additions/tags/entity_type/source_spawner_nbt_blacklist.json
diff --git a/output/resources/ars_additions/.cache/2644c19ed30f4f3dcfaeb5b61eea57fc410bc389 b/output/resources/ars_additions/.cache/2644c19ed30f4f3dcfaeb5b61eea57fc410bc389
index b9c74e3..516609d 100644
--- a/output/resources/ars_additions/.cache/2644c19ed30f4f3dcfaeb5b61eea57fc410bc389
+++ b/output/resources/ars_additions/.cache/2644c19ed30f4f3dcfaeb5b61eea57fc410bc389
@@ -1,3 +1,3 @@
-// 1.21.1	2025-08-24T22:07:42.434667216	Curios for ars_additions
+// 1.21.1	2025-08-24T22:23:42.989118167	Curios for ars_additions
 7d3e0368118f8cf6d43ccc4fb337379e29c14261 data/ars_additions/curios/entities/ars_additions.json
 6530a512bfdefceb97a8c1d6c4a888345d879ff4 data/ars_additions/curios/slots/charm.json
diff --git a/output/resources/ars_additions/.cache/2926e6bb22341b93f95c65f68a5c4c13fe84cb3c b/output/resources/ars_additions/.cache/2926e6bb22341b93f95c65f68a5c4c13fe84cb3c
index d5eb2ae..a404e51 100644
--- a/output/resources/ars_additions/.cache/2926e6bb22341b93f95c65f68a5c4c13fe84cb3c
+++ b/output/resources/ars_additions/.cache/2926e6bb22341b93f95c65f68a5c4c13fe84cb3c
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:07:42.435288715	Languages: en_us for mod: ars_additions
+// 1.21.1	2025-08-24T22:23:42.989873816	Languages: en_us for mod: ars_additions
 333d07c69c0340edcd318ff9d5210e28285874a2 assets/ars_additions/lang/en_us.json
diff --git a/output/resources/ars_additions/.cache/476f8d0317c94dbdd6c0816461b216f6de1bfce0 b/output/resources/ars_additions/.cache/476f8d0317c94dbdd6c0816461b216f6de1bfce0
index 2f396e0..e7096de 100644
--- a/output/resources/ars_additions/.cache/476f8d0317c94dbdd6c0816461b216f6de1bfce0
+++ b/output/resources/ars_additions/.cache/476f8d0317c94dbdd6c0816461b216f6de1bfce0
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:07:42.428942989	Source Spawner Datagen
-820e9d78063cec449ddce1880219061df30fcdfc data/ars_additions/recipe/source_spawner/source_spawner.json
+// 1.21.1	2025-08-24T22:23:42.980394196	Source Spawner Datagen
+b1fa4594bb94b5ac77516023fb9f1d718ccf9778 data/ars_additions/recipe/source_spawner/source_spawner.json
diff --git a/output/resources/ars_additions/.cache/565d4a0e5d73e1b90a97b38801b215c439c0918e b/output/resources/ars_additions/.cache/565d4a0e5d73e1b90a97b38801b215c439c0918e
index 291c70c..3f4160b 100644
--- a/output/resources/ars_additions/.cache/565d4a0e5d73e1b90a97b38801b215c439c0918e
+++ b/output/resources/ars_additions/.cache/565d4a0e5d73e1b90a97b38801b215c439c0918e
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:07:42.434474837	Imbue Spell Scroll Datagen
+// 1.21.1	2025-08-24T22:23:42.988898758	Imbue Spell Scroll Datagen
 e03259ad1fcfe99d7ba1186572e9110c66ec1b2d data/ars_additions/recipe/imbue_spell_scroll/imbue_scroll.json
diff --git a/output/resources/ars_additions/.cache/59eb3dbb5f86130e09b3c62d89b9525ee01cf52d b/output/resources/ars_additions/.cache/59eb3dbb5f86130e09b3c62d89b9525ee01cf52d
index c787a4b..0e33bdb 100644
--- a/output/resources/ars_additions/.cache/59eb3dbb5f86130e09b3c62d89b9525ee01cf52d
+++ b/output/resources/ars_additions/.cache/59eb3dbb5f86130e09b3c62d89b9525ee01cf52d
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.429122904	Loot Tables
+// 1.21.1	2025-08-24T22:23:42.980540449	Loot Tables
 90ca9f74061b62b4c9da088bb033cecb83cd5d4d data/ars_additions/loot_table/blocks/archwood_chain.json
 767f22183d3273de37d51757b0870ad5eb4006df data/ars_additions/loot_table/blocks/archwood_lantern.json
 bf48be6515b6bb9657bd95515eefd64ccad02556 data/ars_additions/loot_table/blocks/archwood_magelight_lantern.json
diff --git a/output/resources/ars_additions/.cache/9af1fd46421fb139d00147218e960b3bffafab98 b/output/resources/ars_additions/.cache/9af1fd46421fb139d00147218e960b3bffafab98
index f1f8c45..fff494a 100644
--- a/output/resources/ars_additions/.cache/9af1fd46421fb139d00147218e960b3bffafab98
+++ b/output/resources/ars_additions/.cache/9af1fd46421fb139d00147218e960b3bffafab98
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:07:42.438891014	Bulk Scribing Datagen
+// 1.21.1	2025-08-24T22:23:42.99042348	Bulk Scribing Datagen
 44b2e3489e110e775ef8dccb424b2c07a0e8a97d data/ars_additions/recipe/bulk_scribing/bulk_scriber.json
diff --git a/output/resources/ars_additions/.cache/9fb1092f32d4fcbf9e061ffd718d4ec689c6c95e b/output/resources/ars_additions/.cache/9fb1092f32d4fcbf9e061ffd718d4ec689c6c95e
index db26dfb..c5c632e 100644
--- a/output/resources/ars_additions/.cache/9fb1092f32d4fcbf9e061ffd718d4ec689c6c95e
+++ b/output/resources/ars_additions/.cache/9fb1092f32d4fcbf9e061ffd718d4ec689c6c95e
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.43904452	Recipes
+// 1.21.1	2025-08-24T22:23:42.990570835	Recipes
 cc12f14a50fcb9fc3534d163bef12157bc93993f data/ars_additions/advancement/recipes/decorations/crafting/stonecutter/cracked_polished_sourcestone_wall_0.json
 03f1a2df568393b13ef95b0aa52124de19dd4974 data/ars_additions/advancement/recipes/decorations/crafting/stonecutter/cracked_sourcestone_wall_0.json
 bd18a5f3e1d32ad12c5813825a753448606993d3 data/ars_additions/advancement/recipes/decorations/crafting/stonecutter/polished_sourcestone_button_0.json
diff --git a/output/resources/ars_additions/.cache/ac35f14390caaf96d25142beb0aa02f71a1445e0 b/output/resources/ars_additions/.cache/ac35f14390caaf96d25142beb0aa02f71a1445e0
index eca79ab..d1840ff 100644
--- a/output/resources/ars_additions/.cache/ac35f14390caaf96d25142beb0aa02f71a1445e0
+++ b/output/resources/ars_additions/.cache/ac35f14390caaf96d25142beb0aa02f71a1445e0
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.440543366	Tags for minecraft:worldgen/structure mod id ars_additions
+// 1.21.1	2025-08-24T22:23:42.992319915	Tags for minecraft:worldgen/structure mod id ars_additions
 8430db5aad8a63c53e57cb241c8b9287047516e7 data/ars_additions/tags/worldgen/structure/ancient_city.json
 1d77e12b29c0993e3de4bcecbbd80bcda4aea903 data/ars_additions/tags/worldgen/structure/bastion.json
 2c546cc2106567730d985ae6d57c9aadb71a8945 data/ars_additions/tags/worldgen/structure/desert_temple.json
diff --git a/output/resources/ars_additions/.cache/af617d50f86666f21b24a8919c1c50aad8faa4f8 b/output/resources/ars_additions/.cache/af617d50f86666f21b24a8919c1c50aad8faa4f8
index aaccbc3..db6d085 100644
--- a/output/resources/ars_additions/.cache/af617d50f86666f21b24a8919c1c50aad8faa4f8
+++ b/output/resources/ars_additions/.cache/af617d50f86666f21b24a8919c1c50aad8faa4f8
@@ -1 +1 @@
-// 1.21.1	2025-08-24T22:07:42.428523517	Imbuement
+// 1.21.1	2025-08-24T22:23:42.980071445	Imbuement
diff --git a/output/resources/ars_additions/.cache/bbc6b206c68cdbed5b9dbf1c7de76d89d56c6f45 b/output/resources/ars_additions/.cache/bbc6b206c68cdbed5b9dbf1c7de76d89d56c6f45
index 104a2b9..78c803a 100644
--- a/output/resources/ars_additions/.cache/bbc6b206c68cdbed5b9dbf1c7de76d89d56c6f45
+++ b/output/resources/ars_additions/.cache/bbc6b206c68cdbed5b9dbf1c7de76d89d56c6f45
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.430666323	Block States: ars_additions
+// 1.21.1	2025-08-24T22:23:42.985141815	Block States: ars_additions
 295924d0f71e6aed8052bed69fa90811de84f096 assets/ars_additions/blockstates/archwood_chain.json
 8a543aa6a70d1b8a39b57e382ab833676d76a5ad assets/ars_additions/blockstates/archwood_lantern.json
 d4810f82798c8f38cddec7e9077adb3041772025 assets/ars_additions/blockstates/archwood_magelight_lantern.json
diff --git a/output/resources/ars_additions/.cache/e027af1ad2da61abea7acd24d8910b665b880c45 b/output/resources/ars_additions/.cache/e027af1ad2da61abea7acd24d8910b665b880c45
index 1a57352..a3d5262 100644
--- a/output/resources/ars_additions/.cache/e027af1ad2da61abea7acd24d8910b665b880c45
+++ b/output/resources/ars_additions/.cache/e027af1ad2da61abea7acd24d8910b665b880c45
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.440967507	Locate Structure Datagen
+// 1.21.1	2025-08-24T22:23:42.992866614	Locate Structure Datagen
 7e483eff82ab79a2213d9145d0cb49705ccdfcb6 data/ars_additions/recipe/locate_structure/ancient_city.json
 bdc9405d2ec9ac61a42e7fac30e9344f476a353b data/ars_additions/recipe/locate_structure/arcane_library.json
 30dc75bc7b683b82d119529ebc065da0dd1caa70 data/ars_additions/recipe/locate_structure/bastion.json
diff --git a/output/resources/ars_additions/.cache/e581a97017dbee0dfb224f8cd0af2a5bd8d029a8 b/output/resources/ars_additions/.cache/e581a97017dbee0dfb224f8cd0af2a5bd8d029a8
index b07d61d..4e50486 100644
--- a/output/resources/ars_additions/.cache/e581a97017dbee0dfb224f8cd0af2a5bd8d029a8
+++ b/output/resources/ars_additions/.cache/e581a97017dbee0dfb224f8cd0af2a5bd8d029a8
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.440211276	Apparatus
+// 1.21.1	2025-08-24T22:23:42.991963601	Apparatus
 a47b79c163a2e20067540dc76e06a5ab75c49edd data/ars_additions/recipe/apparatus/advanced_dominion_wand.json
 2ad7c89701c9623122857bb7cf814607bf180688 data/ars_additions/recipe/apparatus/dispel_protection_charm.json
 273948936e2da5c841690f3e6b53f7e5f48783be data/ars_additions/recipe/apparatus/ender_mask_charm.json
diff --git a/output/resources/ars_additions/.cache/e5c5eb35b4ba40351ecb7d9f04c3527f2f5779b0 b/output/resources/ars_additions/.cache/e5c5eb35b4ba40351ecb7d9f04c3527f2f5779b0
index 41d3a5a..a211d97 100644
--- a/output/resources/ars_additions/.cache/e5c5eb35b4ba40351ecb7d9f04c3527f2f5779b0
+++ b/output/resources/ars_additions/.cache/e5c5eb35b4ba40351ecb7d9f04c3527f2f5779b0
@@ -1,3 +1,3 @@
-// 1.21.1	2025-08-24T22:07:42.440808931	Advancements
+// 1.21.1	2025-08-24T22:23:42.992639691	Advancements
 1de4717eb19c334ebe8246f127a1be9f31ac7e08 data/ars_additions/advancement/create_ruined_portal.json
 2dd92fb8d2606650625da417ba48861b10075da6 data/ars_additions/advancement/find_ruined_portal.json
diff --git a/output/resources/ars_additions/.cache/e7c243f38c2b0774b1bff1a90ede0b19a4077240 b/output/resources/ars_additions/.cache/e7c243f38c2b0774b1bff1a90ede0b19a4077240
index 8b4e0d1..b543657 100644
--- a/output/resources/ars_additions/.cache/e7c243f38c2b0774b1bff1a90ede0b19a4077240
+++ b/output/resources/ars_additions/.cache/e7c243f38c2b0774b1bff1a90ede0b19a4077240
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.434842352	Patchouli
+// 1.21.1	2025-08-24T22:23:42.989355279	Patchouli
 0bb2ffa8cc5faa08b223025808c506e7d7e00abf assets/ars_nouveau/patchouli_books/worn_notebook/en_us/entries/equipment/charms.json
 b071ba4d503178cf0f66cdbcd3d7399a45643d35 assets/ars_nouveau/patchouli_books/worn_notebook/en_us/entries/equipment/handy_haversack.json
 d2c88052e609bae2f8ce074e92826096a08f086a assets/ars_nouveau/patchouli_books/worn_notebook/en_us/entries/equipment/imbued_spell_parchment.json
diff --git a/output/resources/ars_additions/.cache/e870fe4addee80d44a417876877dd9898af6067c b/output/resources/ars_additions/.cache/e870fe4addee80d44a417876877dd9898af6067c
index 89dba0b..6a78e3f 100644
--- a/output/resources/ars_additions/.cache/e870fe4addee80d44a417876877dd9898af6067c
+++ b/output/resources/ars_additions/.cache/e870fe4addee80d44a417876877dd9898af6067c
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.428149569	Tags for minecraft:item mod id ars_additions
+// 1.21.1	2025-08-24T22:23:42.979779691	Tags for minecraft:item mod id ars_additions
 35133e95f1c8fdd7a1c21afcc231fc0bffefb9a8 data/ars_additions/tags/item/forgotten_knowledge.json
 d574457fb59e80f6dba71cbca0dbed807d5d55ee data/ars_additions/tags/item/spellweave_compatible.json
 c417d25dbf09fa35316563dab44086ea9c57398f data/ars_nouveau/tags/item/boot.json
diff --git a/output/resources/ars_additions/.cache/f0f2bcace349d412215f006e6183aa56b9316c3c b/output/resources/ars_additions/.cache/f0f2bcace349d412215f006e6183aa56b9316c3c
index 173383e..d216863 100644
--- a/output/resources/ars_additions/.cache/f0f2bcace349d412215f006e6183aa56b9316c3c
+++ b/output/resources/ars_additions/.cache/f0f2bcace349d412215f006e6183aa56b9316c3c
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:07:42.428719522	Glyph Recipes
+// 1.21.1	2025-08-24T22:23:42.980215483	Glyph Recipes
 e408d53bf4efbff53a8ae0a316692f4f9fd9f607 data/ars_additions/recipe/glyph_mark.json
 45ec09bbb6a7b1838190ebe13c92ac630f6d931b data/ars_additions/recipe/glyph_recall.json
 e6aa20d21e0b2c7666ed3d9ca27c289d3f68354a data/ars_additions/recipe/glyph_retaliate.json
diff --git a/output/resources/ars_additions/data/ars_additions/recipe/source_spawner/source_spawner.json b/output/resources/ars_additions/data/ars_additions/recipe/source_spawner/source_spawner.json
index 85e6321..2e0e7db 100644
--- a/output/resources/ars_additions/data/ars_additions/recipe/source_spawner/source_spawner.json
+++ b/output/resources/ars_additions/data/ars_additions/recipe/source_spawner/source_spawner.json
@@ -1,12 +1,16 @@
 {
   "type": "ars_additions:source_spawner",
-  "entity": {
-    "tag": "ars_additions:source_spawner_nbt_blacklist"
-  },
-  "id": "ars_additions:blacklist",
+  "id": "ars_additions:default",
   "tag_modifiers": [
     {
-      "type": "ars_additions:remove_tag"
+      "type": "ars_additions:remove_guaranteed_drops"
+    },
+    {
+      "type": "ars_additions:remove_tag",
+      "strip_tags": [
+        "Inventory",
+        "Items"
+      ]
     }
   ]
 }
\ No newline at end of file
