diff --git a/output/resources/ars_nouveau/.cache/103d9f3f36b01595f1aa5172191e60eff02e6924 b/output/resources/ars_nouveau/.cache/103d9f3f36b01595f1aa5172191e60eff02e6924
index 8983de2..4bae8e7 100644
--- a/output/resources/ars_nouveau/.cache/103d9f3f36b01595f1aa5172191e60eff02e6924
+++ b/output/resources/ars_nouveau/.cache/103d9f3f36b01595f1aa5172191e60eff02e6924
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.331813172	Registries
+// 1.21.1	2025-08-24T23:03:39.982064581	Registries
 32970dfec52f4cfe787c7ad901a17e7da6e2643a data/ars_nouveau/banner_pattern/abjuration.json
 025338cd2e16245ba0dc81f137084ab153763212 data/ars_nouveau/banner_pattern/air.json
 4f0f3c8e088398e980ad282d91d3ec8b1c59da14 data/ars_nouveau/banner_pattern/conjuration.json
diff --git a/output/resources/ars_nouveau/.cache/1fdcfe145635ded7bf0d505f6787667f0b49b64e b/output/resources/ars_nouveau/.cache/1fdcfe145635ded7bf0d505f6787667f0b49b64e
index 9ebae4e..dfd3a0a 100644
--- a/output/resources/ars_nouveau/.cache/1fdcfe145635ded7bf0d505f6787667f0b49b64e
+++ b/output/resources/ars_nouveau/.cache/1fdcfe145635ded7bf0d505f6787667f0b49b64e
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.32766253	Ars Nouveau's Damage Type Data
+// 1.21.1	2025-08-24T23:03:39.977450026	Ars Nouveau's Damage Type Data
 4b4b2ff422d23e4cb6282f664d89dad7ac8ff8e6 data/ars_nouveau/damage_type/crush.json
 fe8c85729655f971ee94229cbb7d68bbb513d08b data/ars_nouveau/damage_type/flare.json
 3aa319bb73ba25bbeae84aace10603b4900b6100 data/ars_nouveau/damage_type/frost.json
diff --git a/output/resources/ars_nouveau/.cache/2741af456691947b147d613b4e1cc70c9e71dd32 b/output/resources/ars_nouveau/.cache/2741af456691947b147d613b4e1cc70c9e71dd32
index 4a5c432..4ea0a8f 100644
--- a/output/resources/ars_nouveau/.cache/2741af456691947b147d613b4e1cc70c9e71dd32
+++ b/output/resources/ars_nouveau/.cache/2741af456691947b147d613b4e1cc70c9e71dd32
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.327493754	Tags for minecraft:damage_type mod id ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.977252085	Tags for minecraft:damage_type mod id ars_nouveau
 8641b05da5b216be05123e3ed96b606930b31f26 data/minecraft/tags/damage_type/always_hurts_ender_dragons.json
 c07768ed08384f5d693ed8bed719a825a531b78e data/minecraft/tags/damage_type/bypasses_armor.json
 4eb8deadc24e4e017387ef06cfa6543373cea902 data/minecraft/tags/damage_type/is_fall.json
diff --git a/output/resources/ars_nouveau/.cache/2851bad206eacbcf07f44fe5df73fe54d649e042 b/output/resources/ars_nouveau/.cache/2851bad206eacbcf07f44fe5df73fe54d649e042
index 8aa4068..b65e087 100644
--- a/output/resources/ars_nouveau/.cache/2851bad206eacbcf07f44fe5df73fe54d649e042
+++ b/output/resources/ars_nouveau/.cache/2851bad206eacbcf07f44fe5df73fe54d649e042
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.337517885	Ars Nouveau's Jukebox Data
+// 1.21.1	2025-08-24T23:03:39.990621241	Ars Nouveau's Jukebox Data
 7e8e4bde7700db0a53308df777880a129e98b5d0 data/ars_nouveau/jukebox_song/aria_biblio.json
 3a76a49e3c264090a26ae2e512fdf36e94a0e382 data/ars_nouveau/jukebox_song/firel_the_wild_hunt.json
 8df17e7d75fa4d9822e6b2ed109e2c9f92e874e0 data/ars_nouveau/jukebox_song/thistle_the_sound_of_glass.json
diff --git a/output/resources/ars_nouveau/.cache/303313689de900e45df1f7ca17c3d36f2c78ac0c b/output/resources/ars_nouveau/.cache/303313689de900e45df1f7ca17c3d36f2c78ac0c
index f9d9963..41669a7 100644
--- a/output/resources/ars_nouveau/.cache/303313689de900e45df1f7ca17c3d36f2c78ac0c
+++ b/output/resources/ars_nouveau/.cache/303313689de900e45df1f7ca17c3d36f2c78ac0c
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.327080941	Tags for minecraft:worldgen/biome mod id ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.97684872	Tags for minecraft:worldgen/biome mod id ars_nouveau
 92c4e65802ffcb7533e9259f0b0356510154d662 data/ars_nouveau/tags/worldgen/biome/archwood_biome.json
 60daaef808d1b0ed031d9007b5ad3007d1fdb695 data/ars_nouveau/tags/worldgen/biome/berry_spawn.json
 e65d864a8148e6a6c372dcc989f9c68e0ae6110c data/ars_nouveau/tags/worldgen/biome/no_mob_spawn.json
diff --git a/output/resources/ars_nouveau/.cache/366e6cedc97b567a76524805ff813b15bef0b4f4 b/output/resources/ars_nouveau/.cache/366e6cedc97b567a76524805ff813b15bef0b4f4
index 7aea724..ed1d67c 100644
--- a/output/resources/ars_nouveau/.cache/366e6cedc97b567a76524805ff813b15bef0b4f4
+++ b/output/resources/ars_nouveau/.cache/366e6cedc97b567a76524805ff813b15bef0b4f4
@@ -1,3 +1,3 @@
-// 1.21.1	2025-08-24T22:23:58.34310157	Dispel Entity Datagen
+// 1.21.1	2025-08-24T23:03:39.996187097	Dispel Entity Datagen
 55a9fb696041d92f6546f9017307a9e660678713 data/ars_nouveau/recipe/dispel_entity/blaze_powder.json
 31a1694f2ef8f0033e7c4e0440977594edcd4c7d data/ars_nouveau/recipe/dispel_entity/wixie_shard.json
diff --git a/output/resources/ars_nouveau/.cache/43f70b1d0fc3650d25dddf44c6efffef1900bb18 b/output/resources/ars_nouveau/.cache/43f70b1d0fc3650d25dddf44c6efffef1900bb18
index 1775ca6..17819ee 100644
--- a/output/resources/ars_nouveau/.cache/43f70b1d0fc3650d25dddf44c6efffef1900bb18
+++ b/output/resources/ars_nouveau/.cache/43f70b1d0fc3650d25dddf44c6efffef1900bb18
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.342795878	Crush
+// 1.21.1	2025-08-24T23:03:39.995916951	Crush
 b74461afbe445ac042cd1d04298474b2403054ae data/ars_nouveau/recipe/blue_dye.json
 0ba5aafe171e9aab7bec69520d99698d5829fd97 data/ars_nouveau/recipe/brown_dye.json
 dfee25791763eafe3dacdb2ce45173e1f5862a70 data/ars_nouveau/recipe/cobblestone.json
diff --git a/output/resources/ars_nouveau/.cache/457d401e850e914fe39c053c4aa175787ef49774 b/output/resources/ars_nouveau/.cache/457d401e850e914fe39c053c4aa175787ef49774
index 9365f48..1412c77 100644
--- a/output/resources/ars_nouveau/.cache/457d401e850e914fe39c053c4aa175787ef49774
+++ b/output/resources/ars_nouveau/.cache/457d401e850e914fe39c053c4aa175787ef49774
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.342600081	Scry Ritual Datagen
+// 1.21.1	2025-08-24T23:03:39.995711526	Scry Ritual Datagen
 26baa828b81ffa33bb56b9ec47bb3d49fa2a0106 data/ars_nouveau/recipe/scry_ritual/coal_ores.json
 d2dcdf8259d2bc6d29830a4f81b19f5d009cd062 data/ars_nouveau/recipe/scry_ritual/copper_ores.json
 a6665e46a3e3d697fa16e1d7ec8f9a814fb11a0c data/ars_nouveau/recipe/scry_ritual/diamond_ores.json
diff --git a/output/resources/ars_nouveau/.cache/56408d4385cae198e776e71d25785a05a196148d b/output/resources/ars_nouveau/.cache/56408d4385cae198e776e71d25785a05a196148d
index e2805fb..22baa6e 100644
--- a/output/resources/ars_nouveau/.cache/56408d4385cae198e776e71d25785a05a196148d
+++ b/output/resources/ars_nouveau/.cache/56408d4385cae198e776e71d25785a05a196148d
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.32693664	Ars Nouveau's Enchantment Data
+// 1.21.1	2025-08-24T23:03:39.97668355	Ars Nouveau's Enchantment Data
 e86598b47a336ed9c1c12c8c0391cf4d2f771c21 data/ars_nouveau/enchantment/mana_boost.json
 321253ad8e6e58e584e3fe140e7fc39a1f2f11b7 data/ars_nouveau/enchantment/mana_regen.json
 48649708bc7378402370f35ac7d600a31385fa01 data/ars_nouveau/enchantment/reactive.json
diff --git a/output/resources/ars_nouveau/.cache/57a62c95f59bc5868cffb1d040840f0d715117bd b/output/resources/ars_nouveau/.cache/57a62c95f59bc5868cffb1d040840f0d715117bd
index 0523a04..5976473 100644
--- a/output/resources/ars_nouveau/.cache/57a62c95f59bc5868cffb1d040840f0d715117bd
+++ b/output/resources/ars_nouveau/.cache/57a62c95f59bc5868cffb1d040840f0d715117bd
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:23:58.338127166	Summon Ritual Datagen
+// 1.21.1	2025-08-24T23:03:39.99127231	Summon Ritual Datagen
 71d5df7266f203db9b869b49529b50e286ea662d data/ars_nouveau/recipe/summon_ritual/bats.json
diff --git a/output/resources/ars_nouveau/.cache/59eb3dbb5f86130e09b3c62d89b9525ee01cf52d b/output/resources/ars_nouveau/.cache/59eb3dbb5f86130e09b3c62d89b9525ee01cf52d
index 4ea5035..7ce1159 100644
--- a/output/resources/ars_nouveau/.cache/59eb3dbb5f86130e09b3c62d89b9525ee01cf52d
+++ b/output/resources/ars_nouveau/.cache/59eb3dbb5f86130e09b3c62d89b9525ee01cf52d
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.324371958	Loot Tables
+// 1.21.1	2025-08-24T23:03:39.974552067	Loot Tables
 4f70eae621815da8d08d3ea0b9fd479b48bad2d3 data/ars_nouveau/loot_table/blocks/agronomic_sourcelink.json
 8a6d60f2b39de0408c7e4537b53e866a04236d22 data/ars_nouveau/loot_table/blocks/alchemical_sourcelink.json
 8bdff8a0a49a9f3f21a822eb1d1d01a950d6ad1a data/ars_nouveau/loot_table/blocks/alteration_table.json
diff --git a/output/resources/ars_nouveau/.cache/5bcee44b915b4e0fcf0f5d8d83123c25b2271d3e b/output/resources/ars_nouveau/.cache/5bcee44b915b4e0fcf0f5d8d83123c25b2271d3e
index 296949f..3e91b76 100644
--- a/output/resources/ars_nouveau/.cache/5bcee44b915b4e0fcf0f5d8d83123c25b2271d3e
+++ b/output/resources/ars_nouveau/.cache/5bcee44b915b4e0fcf0f5d8d83123c25b2271d3e
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:23:58.333640043	Languages: en_us for mod: ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.986408398	Languages: en_us for mod: ars_nouveau
 75cc07f5893b5cdf174cc5ebaec6d45e2c580feb assets/ars_nouveau/lang/en_us.json
diff --git a/output/resources/ars_nouveau/.cache/5c9e5647706b386caccf0aef27efaa68965030b8 b/output/resources/ars_nouveau/.cache/5c9e5647706b386caccf0aef27efaa68965030b8
index 91a0a48..6f3083a 100644
--- a/output/resources/ars_nouveau/.cache/5c9e5647706b386caccf0aef27efaa68965030b8
+++ b/output/resources/ars_nouveau/.cache/5c9e5647706b386caccf0aef27efaa68965030b8
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.334872411	Tags for minecraft:item mod id ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.987943213	Tags for minecraft:item mod id ars_nouveau
 4bb8f8c6be80e74918cc4b3dbfdcc3bc6fb0c978 data/ars_nouveau/tags/item/apparatus_not_consumed.json
 bde6ca31173d1f22d5f6fe355dc90c9faa35b239 data/ars_nouveau/tags/item/golem/shard.json
 35133e95f1c8fdd7a1c21afcc231fc0bffefb9a8 data/ars_nouveau/tags/item/interact_jar_blacklist.json
diff --git a/output/resources/ars_nouveau/.cache/67d77e74b7ca83296a66c3e6c8d3b1c676ec64ab b/output/resources/ars_nouveau/.cache/67d77e74b7ca83296a66c3e6c8d3b1c676ec64ab
index fe59cd5..bd5c827 100644
--- a/output/resources/ars_nouveau/.cache/67d77e74b7ca83296a66c3e6c8d3b1c676ec64ab
+++ b/output/resources/ars_nouveau/.cache/67d77e74b7ca83296a66c3e6c8d3b1c676ec64ab
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:23:58.324236995	Tags for minecraft:banner_pattern mod id ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.974364255	Tags for minecraft:banner_pattern mod id ars_nouveau
 0795140283c290694f9f2dbdeb4a3a07361643fb data/ars_nouveau/tags/banner_pattern/pattern_item/ars_stencil.json
diff --git a/output/resources/ars_nouveau/.cache/73a0c150702b0db33c4f7a4031c7885e3223a862 b/output/resources/ars_nouveau/.cache/73a0c150702b0db33c4f7a4031c7885e3223a862
index 4ff6766..b124e90 100644
--- a/output/resources/ars_nouveau/.cache/73a0c150702b0db33c4f7a4031c7885e3223a862
+++ b/output/resources/ars_nouveau/.cache/73a0c150702b0db33c4f7a4031c7885e3223a862
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.329562517	Tags for minecraft:mob_effect mod id ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.979467736	Tags for minecraft:mob_effect mod id ars_nouveau
 08d652bf70406a53ea8f4c794d20f054284765b9 data/ars_nouveau/tags/mob_effect/deny_dispel.json
 84bb5fad029e44f32f49e09611492a3f0cede482 data/ars_nouveau/tags/mob_effect/to_sync.json
 0315b9264874191953c091e06cf2d340f81d6d67 data/ars_nouveau/tags/mob_effect/unstable_gifts.json
diff --git a/output/resources/ars_nouveau/.cache/7b0970ef111b287b053eb2a386d0fca6ee5aa291 b/output/resources/ars_nouveau/.cache/7b0970ef111b287b053eb2a386d0fca6ee5aa291
index a27ed19..75a7213 100644
--- a/output/resources/ars_nouveau/.cache/7b0970ef111b287b053eb2a386d0fca6ee5aa291
+++ b/output/resources/ars_nouveau/.cache/7b0970ef111b287b053eb2a386d0fca6ee5aa291
@@ -1,3 +1,3 @@
-// 1.21.1	2025-08-24T22:23:58.327264073	Tags for minecraft:worldgen/placed_feature mod id ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.977005844	Tags for minecraft:worldgen/placed_feature mod id ars_nouveau
 35133e95f1c8fdd7a1c21afcc231fc0bffefb9a8 data/ars_nouveau/tags/worldgen/placed_feature/archwood_trees.json
 35133e95f1c8fdd7a1c21afcc231fc0bffefb9a8 data/ars_nouveau/tags/worldgen/placed_feature/source_berries.json
diff --git a/output/resources/ars_nouveau/.cache/8b3db1e27f2abe56f6016cbf2b3caa59e9d0ff55 b/output/resources/ars_nouveau/.cache/8b3db1e27f2abe56f6016cbf2b3caa59e9d0ff55
index 2406e7b..98a44a6 100644
--- a/output/resources/ars_nouveau/.cache/8b3db1e27f2abe56f6016cbf2b3caa59e9d0ff55
+++ b/output/resources/ars_nouveau/.cache/8b3db1e27f2abe56f6016cbf2b3caa59e9d0ff55
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.334710377	Grab bag Datagen
+// 1.21.1	2025-08-24T23:03:39.987781541	Grab bag Datagen
 9f12aa688a076e34ed4b457c7fe492205a824d83 data/ars_nouveau/recipe/apprentice_book_upgrade.json
 98ef162dd5449b8a1e3b5f9a98cf46852e4d2da3 data/ars_nouveau/recipe/archmage_book_upgrade.json
 09fbe1cf323f738d045cfc231fc12a311ab62325 data/ars_nouveau/recipe/fill_potion_flask_0.json
diff --git a/output/resources/ars_nouveau/.cache/95c82c0d51fb7266bfeb5f2d122be5f9b82ff056 b/output/resources/ars_nouveau/.cache/95c82c0d51fb7266bfeb5f2d122be5f9b82ff056
index 33671fa..f13d175 100644
--- a/output/resources/ars_nouveau/.cache/95c82c0d51fb7266bfeb5f2d122be5f9b82ff056
+++ b/output/resources/ars_nouveau/.cache/95c82c0d51fb7266bfeb5f2d122be5f9b82ff056
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:23:58.333746102	Tags for minecraft:worldgen/structure mod id ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.986578788	Tags for minecraft:worldgen/structure mod id ars_nouveau
 eedfc051e19e8fe8ea38287245711becdf5d7ba7 data/ars_nouveau/tags/worldgen/structure/wilden_den.json
diff --git a/output/resources/ars_nouveau/.cache/9fb1092f32d4fcbf9e061ffd718d4ec689c6c95e b/output/resources/ars_nouveau/.cache/9fb1092f32d4fcbf9e061ffd718d4ec689c6c95e
index fcdd158..70225da 100644
--- a/output/resources/ars_nouveau/.cache/9fb1092f32d4fcbf9e061ffd718d4ec689c6c95e
+++ b/output/resources/ars_nouveau/.cache/9fb1092f32d4fcbf9e061ffd718d4ec689c6c95e
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.338327801	Recipes
+// 1.21.1	2025-08-24T23:03:39.991430026	Recipes
 a76e5d9eef406841f56740535ec2afdd509de507 data/ars_nouveau/advancement/recipes/building_blocks/archwood_button.json
 443515bb80018fbe37a128cb6849f5e818be315d data/ars_nouveau/advancement/recipes/building_blocks/archwood_door.json
 4dfc3afae271aae8423d20dedb67dc601ff71fa0 data/ars_nouveau/advancement/recipes/building_blocks/archwood_fence.json
diff --git a/output/resources/ars_nouveau/.cache/a4c3c262a85f6a179e7e29128e8b31d4da806803 b/output/resources/ars_nouveau/.cache/a4c3c262a85f6a179e7e29128e8b31d4da806803
index e9701ee..fda1158 100644
--- a/output/resources/ars_nouveau/.cache/a4c3c262a85f6a179e7e29128e8b31d4da806803
+++ b/output/resources/ars_nouveau/.cache/a4c3c262a85f6a179e7e29128e8b31d4da806803
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.329770226	Ars Nouveau Item Models
+// 1.21.1	2025-08-24T23:03:39.979651119	Ars Nouveau Item Models
 11ee554b11a3a830e89b742762e5b754c2ecb5ee assets/ars_nouveau/models/item/agronomic_sourcelink.json
 20a434b964b88bc010c17e0616f2c2601b53479d assets/ars_nouveau/models/item/alakarkinos_charm.json
 533b61aabca43719a266cddda3b7ec426135e5e6 assets/ars_nouveau/models/item/alakarkinos_hat.json
diff --git a/output/resources/ars_nouveau/.cache/a5922da861c7ee259705eec42cf3df986f0db576 b/output/resources/ars_nouveau/.cache/a5922da861c7ee259705eec42cf3df986f0db576
index c4e8e11..e6a7499 100644
--- a/output/resources/ars_nouveau/.cache/a5922da861c7ee259705eec42cf3df986f0db576
+++ b/output/resources/ars_nouveau/.cache/a5922da861c7ee259705eec42cf3df986f0db576
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.327799807	Block States: ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.977642837	Block States: ars_nouveau
 1b2e60c29cbc9cba57df1e1d0fd9b6f5394b7980 assets/ars_nouveau/blockstates/agronomic_sourcelink.json
 d445d0443f46e8aade3134187d121e73d66b8568 assets/ars_nouveau/blockstates/alakarkinos_hat.json
 ca9e60aa92a8af0ebf19eb2abd29e68574574b75 assets/ars_nouveau/blockstates/alchemical_sourcelink.json
diff --git a/output/resources/ars_nouveau/.cache/abe0934abf823d33d3e88ae408c1d215f03f6bdc b/output/resources/ars_nouveau/.cache/abe0934abf823d33d3e88ae408c1d215f03f6bdc
index 829c43f..e7b9d12 100644
--- a/output/resources/ars_nouveau/.cache/abe0934abf823d33d3e88ae408c1d215f03f6bdc
+++ b/output/resources/ars_nouveau/.cache/abe0934abf823d33d3e88ae408c1d215f03f6bdc
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:23:58.333487407	Budding Conversion Datagen
+// 1.21.1	2025-08-24T23:03:39.986246615	Budding Conversion Datagen
 bc1af13edf233755680b29b8951ea6dc71e98707 data/ars_nouveau/recipe/budding_conversion/budding_amethyst.json
diff --git a/output/resources/ars_nouveau/.cache/af617d50f86666f21b24a8919c1c50aad8faa4f8 b/output/resources/ars_nouveau/.cache/af617d50f86666f21b24a8919c1c50aad8faa4f8
index c060c7c..284f527 100644
--- a/output/resources/ars_nouveau/.cache/af617d50f86666f21b24a8919c1c50aad8faa4f8
+++ b/output/resources/ars_nouveau/.cache/af617d50f86666f21b24a8919c1c50aad8faa4f8
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.333848082	Imbuement
+// 1.21.1	2025-08-24T23:03:39.986747994	Imbuement
 d8ef9562f603943cf768422fb009c9ec77d7dc65 data/ars_nouveau/recipe/imbuement_abjuration_essence.json
 98efa28dfca47cd27a0e31e89208825c7e320e61 data/ars_nouveau/recipe/imbuement_air_essence.json
 6d57a8685026b987a6012bb592cf26c3dba0e89c data/ars_nouveau/recipe/imbuement_amethyst.json
diff --git a/output/resources/ars_nouveau/.cache/b7d20159a598c550a793f4787d1e0a9f9d0b61c6 b/output/resources/ars_nouveau/.cache/b7d20159a598c550a793f4787d1e0a9f9d0b61c6
index f143d17..44c215e 100644
--- a/output/resources/ars_nouveau/.cache/b7d20159a598c550a793f4787d1e0a9f9d0b61c6
+++ b/output/resources/ars_nouveau/.cache/b7d20159a598c550a793f4787d1e0a9f9d0b61c6
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.334541902	Curios for ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.987594951	Curios for ars_nouveau
 a515afa522bcbf0b8b05736bae63a9af4054cf26 data/ars_nouveau/curios/entities/an_curios.json
 84db829ff6895b514af66069168752d2bc107a02 data/ars_nouveau/curios/slots/an_focus.json
 240e1f476f220fc486f85d0132786a1374faaa46 data/ars_nouveau/curios/slots/belt.json
diff --git a/output/resources/ars_nouveau/.cache/c03ccd0f93606faadfab316dc8aedf1ba96c844a b/output/resources/ars_nouveau/.cache/c03ccd0f93606faadfab316dc8aedf1ba96c844a
index 4dd035b..bee8d18 100644
--- a/output/resources/ars_nouveau/.cache/c03ccd0f93606faadfab316dc8aedf1ba96c844a
+++ b/output/resources/ars_nouveau/.cache/c03ccd0f93606faadfab316dc8aedf1ba96c844a
@@ -1,3 +1,3 @@
-// 1.21.1	2025-08-24T22:23:58.321339139	Tags for minecraft:enchantment mod id ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.972228144	Tags for minecraft:enchantment mod id ars_nouveau
 e654e32b0c3eb36d0912b1f29ad16fa0429c826f data/minecraft/tags/enchantment/non_treasure.json
 e654e32b0c3eb36d0912b1f29ad16fa0429c826f data/minecraft/tags/enchantment/tradeable.json
diff --git a/output/resources/ars_nouveau/.cache/cdae075e2d4d58384fe4d2075f6d5323fe5baa69 b/output/resources/ars_nouveau/.cache/cdae075e2d4d58384fe4d2075f6d5323fe5baa69
index 15eac2c..895d112 100644
--- a/output/resources/ars_nouveau/.cache/cdae075e2d4d58384fe4d2075f6d5323fe5baa69
+++ b/output/resources/ars_nouveau/.cache/cdae075e2d4d58384fe4d2075f6d5323fe5baa69
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.326470728	Ars Nouveau Caster Tomes Datagen
+// 1.21.1	2025-08-24T23:03:39.976216385	Ars Nouveau Caster Tomes Datagen
 ab9cf50e4958592dcee6f5dae6d6bad2e7fe08ab data/ars_nouveau/recipe/tomes/alex_tome.json
 dae41a867bde9258624fe4676128e8140f2e7fe4 data/ars_nouveau/recipe/tomes/arachne_tome.json
 71e004bfa5837d293c6ee00dd9fd99493b9b9aba data/ars_nouveau/recipe/tomes/aurellia_tome.json
diff --git a/output/resources/ars_nouveau/.cache/d0c924a870929ce1baf161c329e85242236b3530 b/output/resources/ars_nouveau/.cache/d0c924a870929ce1baf161c329e85242236b3530
index f3df691..f49269d 100644
--- a/output/resources/ars_nouveau/.cache/d0c924a870929ce1baf161c329e85242236b3530
+++ b/output/resources/ars_nouveau/.cache/d0c924a870929ce1baf161c329e85242236b3530
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.331559677	Tags for minecraft:entity_type mod id ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.981753448	Tags for minecraft:entity_type mod id ars_nouveau
 a958530eef4b0d5ac2f1c7bd552b60e23cb66d9a data/ars_nouveau/tags/entity_type/an_hostile.json
 4d0235b95823f5d315aae7e9f9e484cbde8294a0 data/ars_nouveau/tags/entity_type/animal_summon_blacklist.json
 290c285b1e334d322b8d070b2c6a071114b1d70c data/ars_nouveau/tags/entity_type/berry_blacklist.json
diff --git a/output/resources/ars_nouveau/.cache/d2032e9832fa171affe4a9c0c99fb97f183ff8a6 b/output/resources/ars_nouveau/.cache/d2032e9832fa171affe4a9c0c99fb97f183ff8a6
index e30854b..c8c0985 100644
--- a/output/resources/ars_nouveau/.cache/d2032e9832fa171affe4a9c0c99fb97f183ff8a6
+++ b/output/resources/ars_nouveau/.cache/d2032e9832fa171affe4a9c0c99fb97f183ff8a6
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:23:58.3273852	atlases generator for ars_nouveau
+// 1.21.1	2025-08-24T23:03:39.977132742	atlases generator for ars_nouveau
 0824a6d8558f29057b8af23a64a6dfa5da2ca88b assets/minecraft/atlases/chests.json
diff --git a/output/resources/ars_nouveau/.cache/d784f732a4c9f307e882d11a0eb56178c990bc6f b/output/resources/ars_nouveau/.cache/d784f732a4c9f307e882d11a0eb56178c990bc6f
index 99c31cc..8f63859 100644
--- a/output/resources/ars_nouveau/.cache/d784f732a4c9f307e882d11a0eb56178c990bc6f
+++ b/output/resources/ars_nouveau/.cache/d784f732a4c9f307e882d11a0eb56178c990bc6f
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.332149502	Alakarkinos Recipe Datagen
+// 1.21.1	2025-08-24T23:03:39.98250173	Alakarkinos Recipe Datagen
 c9b4aad4995335ec8b522c2c100de2e2fa349ca1 data/ars_nouveau/recipe/alakarkinos/desert_pyramid.json
 f4f6bc699a6ece68a4258f1debce219b4c214c3a data/ars_nouveau/recipe/alakarkinos/desert_well.json
 033aff52424edeb952e75c7155376625a9704bad data/ars_nouveau/recipe/alakarkinos/ocean_ruins_cold.json
diff --git a/output/resources/ars_nouveau/.cache/e0d3d0b8d9c807675613821fa865a35f707cd83f b/output/resources/ars_nouveau/.cache/e0d3d0b8d9c807675613821fa865a35f707cd83f
index 849f8b3..018af99 100644
--- a/output/resources/ars_nouveau/.cache/e0d3d0b8d9c807675613821fa865a35f707cd83f
+++ b/output/resources/ars_nouveau/.cache/e0d3d0b8d9c807675613821fa865a35f707cd83f
@@ -1,2 +1,2 @@
-// 1.21.1	2025-08-24T22:23:58.337675149	Data Maps
+// 1.21.1	2025-08-24T23:03:39.990789376	Data Maps
 835362efe6b3a85abe6e05e01dcb8d34823e18a0 data/neoforge/data_maps/item/compostables.json
diff --git a/output/resources/ars_nouveau/.cache/e581a97017dbee0dfb224f8cd0af2a5bd8d029a8 b/output/resources/ars_nouveau/.cache/e581a97017dbee0dfb224f8cd0af2a5bd8d029a8
index 435f63c..20a37ca 100644
--- a/output/resources/ars_nouveau/.cache/e581a97017dbee0dfb224f8cd0af2a5bd8d029a8
+++ b/output/resources/ars_nouveau/.cache/e581a97017dbee0dfb224f8cd0af2a5bd8d029a8
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.332311736	Apparatus
+// 1.21.1	2025-08-24T23:03:39.982690704	Apparatus
 54b2918a0b959ae3e055dae2681322a8800177a2 data/ars_nouveau/recipe/alakarkinos_charm.json
 adc3e3b582ab4d1fd6f29537bddaaecb9eaa0c2d data/ars_nouveau/recipe/alchemists_crown.json
 9a7fefa487b159935c1f620dc9516a6600fb0d4b data/ars_nouveau/recipe/amulet_of_mana_boost.json
diff --git a/output/resources/ars_nouveau/.cache/e5c5eb35b4ba40351ecb7d9f04c3527f2f5779b0 b/output/resources/ars_nouveau/.cache/e5c5eb35b4ba40351ecb7d9f04c3527f2f5779b0
index 7d38164..feb4f79 100644
--- a/output/resources/ars_nouveau/.cache/e5c5eb35b4ba40351ecb7d9f04c3527f2f5779b0
+++ b/output/resources/ars_nouveau/.cache/e5c5eb35b4ba40351ecb7d9f04c3527f2f5779b0
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.343271849	Advancements
+// 1.21.1	2025-08-24T23:03:39.996320517	Advancements
 48dd44d88536431cef54a84d4a7dce06d6567a5d data/ars_nouveau/advancement/alteration_table.json
 da10d6cfc3154e84e349a4f51c6cb4b1bdd9cb97 data/ars_nouveau/advancement/amethyst_golem_charm.json
 576e3c90dbff03f0668db9bce1d6d88ed511e672 data/ars_nouveau/advancement/apprentice_spell_book.json
diff --git a/output/resources/ars_nouveau/.cache/e7c243f38c2b0774b1bff1a90ede0b19a4077240 b/output/resources/ars_nouveau/.cache/e7c243f38c2b0774b1bff1a90ede0b19a4077240
index 18c9a56..2227eca 100644
--- a/output/resources/ars_nouveau/.cache/e7c243f38c2b0774b1bff1a90ede0b19a4077240
+++ b/output/resources/ars_nouveau/.cache/e7c243f38c2b0774b1bff1a90ede0b19a4077240
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.335549479	Patchouli
+// 1.21.1	2025-08-24T23:03:39.98860835	Patchouli
 2ec536fea831588201432d61d70af61385882054 assets/ars_nouveau/patchouli_books/worn_notebook/en_us/entries/armor/alteration_table.json
 a82f4fc73946d228b1e3f5e03d0ecef46f33cdfb assets/ars_nouveau/patchouli_books/worn_notebook/en_us/entries/armor/applying_perks.json
 8ce1d8e886b45066bc5c1d9d8e87bdabbb27ae27 assets/ars_nouveau/patchouli_books/worn_notebook/en_us/entries/armor/armor.json
diff --git a/output/resources/ars_nouveau/.cache/eac00f78ae0d33a31d4f208cb05939a004e6cb91 b/output/resources/ars_nouveau/.cache/eac00f78ae0d33a31d4f208cb05939a004e6cb91
index 9e3945e..3e1214e 100644
--- a/output/resources/ars_nouveau/.cache/eac00f78ae0d33a31d4f208cb05939a004e6cb91
+++ b/output/resources/ars_nouveau/.cache/eac00f78ae0d33a31d4f208cb05939a004e6cb91
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.334015797	AN tags
+// 1.21.1	2025-08-24T23:03:39.986963128	AN tags
 5823b6fde732b358b200753a53df36243488a571 data/ars_nouveau/tags/block/an_decorative.json
 eba19304fe4b26196ee775954d8df6341df49a5a data/ars_nouveau/tags/block/an_false_occlusion.json
 de95bc21249101b079255a1a00274f60d55e0602 data/ars_nouveau/tags/block/blazing_logs.json
diff --git a/output/resources/ars_nouveau/.cache/f0f2bcace349d412215f006e6183aa56b9316c3c b/output/resources/ars_nouveau/.cache/f0f2bcace349d412215f006e6183aa56b9316c3c
index 3381c1d..55565e5 100644
--- a/output/resources/ars_nouveau/.cache/f0f2bcace349d412215f006e6183aa56b9316c3c
+++ b/output/resources/ars_nouveau/.cache/f0f2bcace349d412215f006e6183aa56b9316c3c
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.323019165	Glyph Recipes
+// 1.21.1	2025-08-24T23:03:39.973191929	Glyph Recipes
 231d9f126c606465601e36e83948dda99a508834 data/ars_nouveau/recipe/glyph_accelerate.json
 dbad772acd31382cfcf19c7f276da5f91354dfcf data/ars_nouveau/recipe/glyph_amplify.json
 1a8499b67d1ce8614ef1d420106d32d27567eeb9 data/ars_nouveau/recipe/glyph_animate_block.json
diff --git a/output/resources/ars_nouveau/.cache/f49682cfc17456297bb895b136258055d85afc0c b/output/resources/ars_nouveau/.cache/f49682cfc17456297bb895b136258055d85afc0c
index 3ad1fce..fbdbd31 100644
--- a/output/resources/ars_nouveau/.cache/f49682cfc17456297bb895b136258055d85afc0c
+++ b/output/resources/ars_nouveau/.cache/f49682cfc17456297bb895b136258055d85afc0c
@@ -1,4 +1,4 @@
-// 1.21.1	2025-08-24T22:23:58.337814119	ArsNouveau: Json Datagen
+// 1.21.1	2025-08-24T23:03:39.990955306	ArsNouveau: Json Datagen
 e59d0832c1d168df4772338fe24485c4b6b94723 data/ars_nouveau/recipe/dye_apprentice_spell_book.json
 d89b9456df55cd4bd1c30fa9fa253ac31fc7bf37 data/ars_nouveau/recipe/dye_arcanist_boots.json
 9e0bcbe43d8ea6f0c1d354eb28878c8053ebe006 data/ars_nouveau/recipe/dye_arcanist_hood.json
